parley/src/layout/editor.rs

```diff
@@ -442,6 +464,18 @@ where
         self.buffer.clone().into()
     }
 
+    /// Get the current generation of the layout, to decide whether to draw
+    pub fn generation(&self) -> usize {
```

Comment:
Can we make this an opaque value which is only `PartialEq` (with an `as_raw` method).

Users would probably be tempted to perform a `>` check, but that's invalid (presuming that we decide performing more than $$2^{32}$$ operations is allowed)

Reply:
You're right, some people are bonkers. :+ )